packages/app/src/runner/studio/mounter.ts

```diff
@@ -5,14 +5,7 @@ import AssertionOptions from './AssertionOptions.ce.vue'
 import { getOrCreateHelperDom, getSelectorHighlightStyles } from '../dom'
```

Comment:
This is just a straight refactor + adding types. None of the logic should be changed here.